edit/edit.go

```diff
@@ -290,6 +290,45 @@ func where(ed *Editor, a Address) (addr, error) {
 	return at, nil
 }
 
+// WhereLine returns the line number corresponding to the start and
+// end of the address.
+// Dot is set to the address.
+func (ed *Editor) WhereLine(a Address) (lineFrom, lineTo int64, err error) {
```

Comment:
Big shrugs. I'll stick with int64 barring another more compelling reason to stick with just ints.